nmmo/core/env.py

```diff
@@ -88,12 +78,6 @@ def box(rows, cols):
     if self.config.PROVIDE_ACTION_TARGETS:
       obs_space['ActionTargets'] = self.action_space(None)
 
-    if self._task_encoding:
```

Comment:
don't we need this somewhere?

Reply:
We need it, but not necessary in this format. Let me check back after seeing the syllabus integration.